tabpy-server/tabpy_server/handlers/base_handler.py

```diff
@@ -47,23 +53,230 @@ def _add_CORS_header(self):
         origin = self.tabpy_state.get_access_control_allow_origin()
         if len(origin) > 0:
             self.set_header("Access-Control-Allow-Origin", origin)
-            logger.debug("Access-Control-Allow-Origin:{}".format(origin))
+            logger.debug(self.append_request_context(
+                "Access-Control-Allow-Origin:{}".format(origin)))
```

Comment:
we should move to f formatting instead of .format for new changes

Reply:
agreed, will update with time